### DISCUSSION TITLE
Why is nextjs using amp-experiment-1.0 instead of amp-experiment-0.1?

### DISCUSSION THREAD

**Discussion by vladbesson** [vercel/next.js#20272] [Help]
Nextjs automatically include `<script async custom-element="amp-experiment" src="https://cdn.ampproject.org/v0/amp-experiment-1.0.js"></script>` to the page when I using amp-experiment tag on the page, but why amp-experiment-1.0 version? Because of that, I got this error — `[amp-experiment] Experiment amp-experiment-1.0 is not enabled` because 1.0 version [is not stable and not ready for production](https://github.com/ampproject/amphtml/issues/21705).
Also another link — [Pin amp-experiment to v0.1](https://github.com/ampproject/amp-wp/pull/4690)

**Comment by c0b41** [comment]
nextjs using [@ampproject/toolbox-optimizer](https://github.com/ampproject/amp-toolbox) maybe ask them
